src/Hangfire.SqlServer/SqlServerJobQueueMonitoringApi.cs

```diff
@@ -61,21 +62,40 @@ public IEnumerable<string> GetQueues()
             }  
         }
 
-        public IEnumerable<int> GetEnqueuedJobIds(string queue, int @from, int perPage)
+        public IEnumerable<int> GetEnqueuedJobIds(string queue, int @from, int perPage, Pager pager = null)
         {
+            dynamic parameters = null;
+            if (pager != null) parameters = new
```

Comment:
use braces around the if as it looks a bit confusing having the braces on the next line when new'ing up a new object